sql/core/src/main/scala/org/apache/spark/sql/DataFrameReader.scala

```diff
@@ -467,6 +467,14 @@ class DataFrameReader private[sql](sparkSession: SparkSession) extends Logging {
    * <li>`pathGlobFilter`: an optional glob pattern to only include files with paths matching
    * the pattern. The syntax follows <code>org.apache.hadoop.fs.GlobFilter</code>.
    * It does not change the behavior of partition discovery.</li>
+   * <li>`modifiedBefore`: an optional timestamp to only include files with
+   * modification times  occurring before the specified time.  The provided timestamp
+   * must be in the following form:  <code>YYYY-MM-DDTHH:mm:ss</code>  Example:
```

Comment:
ditto: Could you use a single space in `form:  <code>`?

Reply:
Done